### PR TITLE
Template!

### PR DESCRIPTION
**Pull request by MoOx**
Thoughts on this? It looks me some times but probably missed a few things.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Please note that an issue must include the information
+that is marked as REQUIRED below, or it may be closed.
+
+**Great Bug Reports** tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can. Try to includes sample code that *anyone* could run to reproduce what I was seeing
```
Comment (cknitt) [inline .github/ISSUE_TEMPLATE/bug.md]: include

what you were seeing

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Changelog of `@reason-react-native/__template__`
+
+## X.Y.Z - 20YY-MM-DD
+
+Short & conscice description
```
Comment (cknitt) [inline CHANGELOG.md]: concise

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- Something, in [1234567](<link to commit on github>) by @<usename>
+
+### 🐛 Fixes
+
+- Something else, in [1234567](<link to commit on github>) by @<usename>
+
+### Internal changes
+
+(If worth mentionning)
```
Comment (cknitt) [inline CHANGELOG.md]: mentioning

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+- The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+- Trolling, insulting/derogatory comments, and personal or political attacks
+- Public or private harassmentthe
```
Comment (cknitt) [inline CODE_OF_CONDUCT.md]: harassment

File: README.md [hunk 5]
```diff
@@ -1,2 +1,95 @@
...
-ReasonML / BuckleScript bindings for ORIGINAL_PACKAGE_NAME
+**How to use this template**
+
+- Put your bindings in `src/ReactNativeSomething` & rename accordingly or use
+  `bsconfig.json` `"namespace"` field (more on this below),
+- Update all occurences of
+
+  - `@reason-react-native/__template__`
+  - `https://github.com/reason-react-native/__template__`
+  - `@react-native-community/something`
```
Comment (cknitt) [inline README.md]: And more importantly `@reason-react-native/something`.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,95 @@
...
+
+---
+
+# `@reason-react-native/__template__`
+
+[![Build Status](https://github.com/reason-react-native/reason-react-native/workflows/Build/badge.svg)](https://github.com/reason-react-native/reason-react-native/actions)
+[![Version](https://img.shields.io/npm/v/@reason-react-native/something.svg)](https://www.npmjs.com/@reason-react-native/something)
+[![Chat](https://img.shields.io/discord/235176658175262720.svg?logo=discord&colorb=blue)](https://reasonml-community.github.io/reason-react-native/discord/)
+
+ReasonML / BuckleScript bindings for
```
Comment (cknitt) [inline README.md]: link to Reason/BuckleScript homepages?

File: package.json [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    "re:build": "bsb -make-world",
+    "re:clean-build": "bsb -clean-world -make-world",
+    "start": "yarn re:start",
+    "build": "yarn re:build",
+    "test": "yarn re:clean-build"
+  },
+  "devDependencies": {
+    "bs-platform": "^5.0.4",
+    "husky": "^1.3.1",
+    "lerna": "^3.4.0",
```
Comment (cknitt) [inline package.json]: Why?

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (MoOx) [reply .github/ISSUE_TEMPLATE/bug.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (MoOx) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (MoOx) [reply CHANGELOG.md]: fixed

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (MoOx) [reply CODE_OF_CONDUCT.md]: fixed

File: README.md (hunk 5 above)
Comment (MoOx) [reply README.md]: I have modified to have `@reason-react-native/__template__` (bindings) & `@react-native-community/something` (original package) only

File: README.md (hunk 6 above)
Comment (MoOx) [reply README.md]: Done.

File: package.json (hunk 7 above)
Comment (MoOx) [reply package.json]: fail :D, fixed

File: README.md (hunk 5 above)
Comment (cknitt) [reply README.md]: The build status still refers to reason-react-native/reason-react-native though.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+[open a new issue](https://github.com/reason-react-native/__template__/issues/new)
+& follow the instructions that should be prefilled for
+[bug](.github/ISSUE_TEMPLATE/bug.md) or
+[feature request](.github/ISSUE_TEMPLATE/feature.md).
+
+People _love_ thorough bug reports. Not even kidding.
+
+## Fixing bugs & implementing features
+
+We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), so
```
Comment (cknitt) [inline CONTRIBUTING.md]: Sorry, one more:

use (lowercase)

File: README.md (hunk 5 above)
Comment (MoOx) [reply README.md]: fixed

**Pull request merged**